examples/radiotoolbutton.py

```diff
@@ -0,0 +1,40 @@
+from gi.repository import Gtk
+
+from sugar3.graphics.radiotoolbutton import RadioToolButton
+from sugar3.graphics.radiopalette import RadioPalette, RadioMenuButton
+from sugar3.graphics.xocolor import XoColor
+import common
+
+test = common.Test()
+test.show()
+
+box = Gtk.HBox()
+test.pack_start(box, True, True, 0)
+box.show()
+
+def echo(button, label):
+    if not button.props.active:
+        return
+    print label
+
+palette = RadioPalette()
+
+button1 = RadioToolButton(icon_name='document-save', accelerator="<ctrl>S", xo_color=XoColor("#FF0000,#00FF00"), tooltip="save")
```

Comment:
Please don't hardcode XoColor values.  Use one from graphics.style or just use the default